test/controllers/HomeControllerSpec.scala

```diff
@@ -25,88 +25,51 @@ import play.api.mvc.{ AnyContentAsEmpty, DefaultActionBuilder, DefaultMessagesAc
 import play.api.test._
 import play.api.test.Helpers._
 import uk.gov.nationalarchives.omega.editorial.controllers.{ HomeController, SessionKeys }
-import support.BaseSpec
+import support.BaseControllerSpec
 
 /** Add your spec here. You can mock out a whole application including requests, plugins etc.
   *
   * For more information, see https://www.playframework.com/documentation/latest/ScalaTestingWithScalaTest
   */
-class HomeControllerSpec extends BaseSpec {
+class HomeControllerSpec extends BaseControllerSpec {
 
   val landingPagePath = "/edit-set/1"
   val loginPagePath = "/login"
 
```

Comment:
There were several scenarios removed:

- `"render the index page from the application"`
- `"render the index page from the router"`
- `"redirect to the login page from the router when requested with invalid session token"`

I believe it was an implied requirement that we keep all the current ones, and not add any new.

Reply:
As it was specified to convert the tests into unit tests, I left only the tests that test the HomeController. The other two scenarios *from the application, and *from the router, are integration tests within the play framework and this is why I removed them. I might have misunderstood the requirements but as a unit test, the Home Controller can only do two things:

a) check the session token, if it's valid, redirect to the landing page.
b) check the session token, if it's invalid, redirect to the login page.

Reply:
I agree with you.

However, this might be contrary to what @rwalpole expected.

Reply:
@tna-erasmos I would say @enriquedelpino has understood the requirement exactly. We are not interested in testing the application or the router for this unit test, so it is fine to remove those tests.

Reply:
I misunderstood, then.

I'll keep this in mind for [this one](https://national-archives.atlassian.net/browse/PACT-815).